src/wt/main.js

```diff
@@ -1,5 +1,33 @@
+import { Worker } from "node:worker_threads";
+import path from "node:path";
+
 const performCalculations = async () => {
-    // Write your code here
+    const workerPath = path.resolve(import.meta.dirname, "worker.js");
+    const resultsCount = 3;
```

Comment:
here should be `const resultsCounts = os.cpus().length`

missed this part from the assignment 🫠